spec/helpers/item_seasons_helper_spec.rb

```diff
@@ -3,6 +3,10 @@
 require 'rails_helper'
 
 RSpec.describe ItemSeasonsHelper, type: :helper do
+  let(:item_season_jan) { ItemSeason.create(produce_item_id: 1, month_index: 1, country_code: 'br') }
+  let(:item_season_feb) { ItemSeason.create(produce_item_id: 1, month_index: 2, country_code: 'br') }
+  let(:item_season_dec) { ItemSeason.create(produce_item_id: 1, month_index: 12, country_code: 'br') }
```

Comment:
**issue:** we should move this closer to where we need it. A suggestion of mine would be create a context:
```rb
context 'when navigating between seasons' do
  let(:item_season_jan) { ... }

  describe '#month_navigation_previous'

  describe '#month_navigation_next'

  # etc
end
```